account_invoice_mass_sending/README.rst

```diff
@@ -0,0 +1,77 @@
+============================
+Account Invoice Mass Sending
+============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--invoicing-lightgray.png?logo=github
+    :target: https://github.com/OCA/server-invoicing/tree/12.0/account_invoice_mass_sending
+    :alt: OCA/server-invoicing
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/server-invoicing-12-0/server-invoicing-12-0-account_invoice_mass_sending
+    :alt: Translate me on Weblate
+
+|badge1| |badge2| |badge3| |badge4| 
+
+Send invoices in mass mode.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+On the invoice list view, select the invoice to send and click on 'Action > Send & print'.
+
+It will used the existing 'SEND & PRINT' action on the invoice with the default value.
+
```

Comment:
@RobinetDenisAcsone You can add a configuration section explaining that there are two channels to configure.